docs/en/rst/installing/mysql.rst

```diff
@@ -3,7 +3,12 @@
 MySQL
 #####
 
-You need MySQL version 5.0.15 or higher.
+We recommend MariaDB, since it's compatible with MySQL, and it's more
+compatible with Bugzilla. If you instead want to use MySQL, you will
+need MySQL version 5.0.15 or higher, but not MySQL 8 or higher.
+
+A large part of this page mentions MySQL, but it doesn't matter if you
+use MariaDB or MySQL, the commands are gonna be the same.
```

Comment:
\<grammar cop\> the commands are **going to** be the same. \</grammarcop\>

Reply:
still waiting on this one.

Reply:
ok, guess we're not still waiting on this one.  Git's "show me the changes since this was requested" didn't show it, but it's removed in the last commit and gone in the overall changes.